src/shared/clients/ecsClient.ts

```diff
@@ -5,8 +5,14 @@
 
 import { ECS } from 'aws-sdk'
 import { ext } from '../extensionGlobals'
+import { getLogger } from '../logger'
 import { ClassToInterfaceType } from '../utilities/tsUtils'
 
+export interface ListServicesResponse {
```

Comment:
doesnt the aws sdk already define something like this? why redefine it here?

Reply:
This is here because information from two different sdk calls needs to be returned. I have renamed it so that it doesn't get confused with the sdk response type.